_rules/text-contrast-afw4f7.md

```diff
@@ -33,7 +33,7 @@ acknowledgments:
 
 This rule applies to any [visible][] character in a [text node][] that is a [child][] in the [flat tree][] of an [HTML element][], except if the [text node][] has an [ancestor][] in the [flat tree][] for which at least one of the following is true:
 
-- **widget**: the ancestor is a [inheriting semantic][] `widget`; or
+- **disabled widget**: the ancestor is a [inheriting semantic][] `widget` that is [disabled][]; or
 - **disabled label**: the ancestor is used in the [accessible name][] of a [inheriting semantic][] `widget` that is [disabled][]; or
 - **disabled group**: the ancestor has a [semantic role][] of `group` and is [disabled][].
```

Comment:
Also realising that the `disabled` bit is now part of all three conditions and could be refactored:

```markdown
This rule applies to any [visible][] character in a [text node][] that is a [child][] in the [flat tree][] of an [HTML element][], except if the [text node][] has an [ancestor][] in the [flat tree][] which is [disabled][] and for which at least one of the following is true:

- **disabled widget**: the ancestor is a [inheriting semantic][] `widget`; or
- **disabled label**: the ancestor is used in the [accessible name][] of a [inheriting semantic][] `widget`; or
- **disabled group**: the ancestor has a [semantic role][] of `group`.
```
Maybe the conditions would then need to be renamed simply `**widget**`, `**label**` and `**group**` 🤔 

Same can be done for the other rule.

We could also the widget/group conditions that are very similar ("the ancestor is a [inheriting semantic][] `group` or `widget`")

This is a bit unrelated, definitely not blocking this PR and can be tackled later if wanted.